doc/src/manual/faq.md

```diff
@@ -818,9 +818,9 @@ some ideas that might help to understand Julia's definition.
   the product of the size. The size of a zero-dimensional array is `()`, and
   therefore its length is `1`.
 * Zero-dimensional arrays don't natively have any dimensions into which you
-  index -- they’re just `A[]`. We can apply the same "trailing one" rule for them
-  as for all other array dimensionalities, so you can indeed index them as
-  `A[1]`, `A[1,1]`, etc.
+  index -- they’re just `A[]`. We can apply the same "trailing one" rule for them 
```

Comment:
```suggestion
  index -- they’re just `A[]`. We can apply the same "trailing one" rule for them
```

Reply:
This suggestion still needs to be committed (and is why Travis & AppVeyor are still failing).